react/package-lock.json

```diff
@@ -0,0 +1,2061 @@
+{
```

Comment:
what is that file vs yarn.lock, which should not have changed much from you just adding 1 new package...